docs/getting-started.md

```diff
@@ -223,10 +224,14 @@ bazel run -c dbg --run_under="valgrind --tool=callgrind --callgrind-out-file=/tm
 ```
 This will output the file at the _absolute_ path given via the `--callgrind-out-file` argument. This file can then be viewed using `kcachegrind` (example: `kcachegrind /tmp/profile.callgrind`), giving lots of useful information about where time is being spent in the code.
 
+## Building for Jetson Nano 
+
+To build for the Jetson Nano, build the target with the `--cpu=jetson_nano` flag and the toolchain will automatically build using the ARM toolchain for Jetson Nano. For example, `bazel build --cpu=jetson_nano //software/geom/...`.
+
 ## Flashing And Debugging A STM32F4 MCU
 1. Make sure you've followed [Installing Firmware Dependencies](#installing-firmware-dependencies), and have a STM32F4 based main board plugged into your computer. Do not plug both the dongle and the robot at the same time!
 2. Make sure the robot is elevated, with the wheels not touching any surface to avoid experimental firmware causing accidental damage.
-3. From the `src` folder, to flash the radio dongle, run `bazel run --cpu=stm32f4 //firmware/tools:legacy_flash_firmware robot`
+3. From the `src` folder, to flash the radio dongle, run `bazel run --cpu=stm32f4 //firmware/tools:legacy\_flash\_firmware robot`
```

Comment:
you don't need the escape \ when between ``, it seems to be rendering them in

Reply:
ah vim was complaining and I got confused whoops